ui/components/app/srp-input/srp-input.js

```diff
@@ -36,8 +36,9 @@ export default function SrpInput({ onChange, srpText }) {
     (newDraftSrp) => {
       let newSrpError = '';
       const joinedDraftSrp = newDraftSrp.join(' ');
```

Comment:
Now that I think of it, can we avoid needing `hasPopulated` at all, and simply do:

```
const joinedDraftSrp = newDraftSrp.join(' ').trim(); // removes extra spacing at the end, so if empty, the string is just `''`
```

Then check `joinedDraftSrp.length` instead?